docs/index.md

```diff
@@ -74,7 +74,7 @@ with server.auth.sign_in(tableau_auth):
     print("\nThere are {} datasources on site: ".format(pagination_item.total_available))
     print([datasource.name for datasource in all_datasources])
 ```
-For Tableau online TSC.TableauAuth is used with site_id
+For Tableau online TSC.TableauAuth should be used with site_id
```

Comment:
How about "To authenticate to Tableau Online, or a site other than the default within a local Tableau Server installation, the site_id parameter must be used to specify the site" just to be clear.  Otherwise, looks good!

Reply:
Suggest:

```tableau_auth = TSC.TableauAuth('USERNAME', 'PASSWORD', site_id='SITE_NAME')
server = TSC.Server('http://SERVER_URL')
with server.auth.sign_in(tableau_auth):
    all_datasources, pagination_item = server.datasources.get()
    print("\nThere are {} datasources on site: ".format(pagination_item.total_available))
    print([datasource.name for datasource in all_datasources])```

```site_id``` can be omitted for singing in to the default site of a local Tableau Server installation.